packages/forms/src/styled/input-group/StyledInputGroup.ts

```diff
@@ -79,6 +80,10 @@ const itemStyles = (props: ThemeProps<DefaultTheme>) => {
       z-index: -1;
     }
 
+    & > ${StyledTextInput}:first-child {
```

Comment:
```suggestion
    & > *:first-child {
```

This override is important no matter what kind of component (`Input`, `Button`) is first.